### DISCUSSION TITLE
Is there minimum IAM policy for working on AWS

### DISCUSSION THREAD

**Discussion by pomcho555** [serverless/serverless#11067] [Q&A] [answered]
I developed CICD pipeline on GitLab CI, and a role on GitLab Runner deploys cloud resources using serverless framework. For security reason, I cannot attach strong policy like `AdministoratorAccess`. So I am finding some kinds of template of IAM policy to work serverless framework in minimum permission. Does somebody already have it?

**Comment by le2sky** [comment, marked as answer]
hello @pomcho555 !🙋‍♂️
i'm having the same problem as you and i found something
perhaps you can check the minimum permission for sls deploy by referring to this.
https://github.com/serverless/serverless/issues/1439
